README.md

```diff
@@ -28,3 +28,127 @@ After matching the slide filenames against the sample IDs from the clinical data
 Regions of interest can be extracted using the python script generate_rois.py. This script consumes a tab-delimited text file describing the whole-slide image files, ROI coordinates, desired size and magnification for extracted ROIs, and then generates a collection of ROI .png images. These images are transformed into a binary for model training and testing by the software described below.
 
 *Note: region extraction depends on the [OpenSlide](http://openslide.org/) library. This library can be installed locally, and is also available in the Docker image described below.*
+
+# Executables for training and testing models
+
+The Docker container provides executables for training SCNN/GSCNN models and for evaluating the accuracy of these models. Both of these executables consume:
+* ROI .png files
+* A .csv file describing the patient IDs, event times, censoring (0 for uncensored, 1 for right-censored), patient indexes (integer indexes assigned to each patient), and any genomic variables. **_Note that the columns for genomic data should be organized in a way that the first coloumns are allocated for those which will NOT get normalized and the latter ones allocated for those that have to get normnalized._** The default normalization method in our model is Z score normalization. An example file is provided in this repository.
+
+## Training
+
+The command
+```
+python model_train.py
+```
+will train a model using the default hyperparameters and data locations described below.
+
+The input parameters and their default values can be listed using
+```
+python model_train.py -h
+```
+that generates output
+```
+usage: model_train.py [-h] [-m M] [-p P] [-f F] [-i I] [-r R] [-t T] [-d D]
+                      [--lr LR] [--me ME] [--kp KP] [--bs BS] [--ic IC]
+                      [--ngf NGF] [--gn GN] [--ns NS] [--nm NM]
+
+Arguments for Training the SCNN/GSCNN model
+
+optional arguments:
+  -h, --help  show this help message and exit
+  -m M        image_only or image_genome; image_only for SCNN and image_genome
+              for GSCNN. The default is image_only.
+  -p P        Path to the Train/Test patient list. The default path is ./inputs/ImageSplits.txt
```

Comment:
Do we use this anymore? I don't think so. The -i argument seems to take care of this.

Reply:
Right, I eliminated it from the arguments list.